pkg/console/starter/starter.go

```diff
@@ -107,6 +110,9 @@ func RunOperator(clientConfig *rest.Config, stopCh <-chan struct{}) error {
 		oauthinformers.WithTweakListOptions(tweakOAuthListOptions),
 	)
 
+	// TODO: Replace this with real event recorder (use ControllerContext).
+	recorder := ctx.EventRecorder
```

Comment:
I need to look into this.  Does this swallow events (not a real recorder)?

Reply:
Maybe add a bit more context to the comment here, quick explain how it differs from a "real" event recorder.

Reply:
This one looks like the real thing.

Reply:
twas my thinking as well :)